vhdbuilder/packer/install-dependencies.sh

```diff
@@ -530,13 +530,13 @@ done
 
 ADDON_IMAGES="
 mcr.microsoft.com/oss/open-policy-agent/gatekeeper:v2.0.1
-mcr.microsoft.com/oss/open-policy-agent/gatekeeper:v3.1.0-beta.8
+mcr.microsoft.com/oss/open-policy-agent/gatekeeper:v3.1.0
```

Comment:
can you move it down below the beta.12? we assume the lower ones are newer

Reply:
@robbiezhang Thanks, updated the order.